WIP fleet management documentation (#705)
neo4j/docs-aura: modules/ROOT/pages/fleet-management/graph-view.adoc

[[sm-deployment-graph]]
= Graph view
:description: This page describes the deployment graph view

> In the graph view, the topology data is displayed as a graph. The graph shows the servers, databases and database instances in the deployment.

Reviewer: ```suggestion
In the graph view, the topology data is displayed as a graph. 
The graph shows the servers, databases, and database instances in the deployment.
```
Instances...

> Depending on whether the view is accessed from *Instances* or *Deployment details*, it will show either all deployments or just the selected deployment.

Reviewer: ```suggestion
Depending on whether the view is accessed from *Instances* or *Deployment details*, it shows either all deployments or just the selected deployment.
```

[.shadow]
image::deployment-graph.png[]

== Legend

> To the right of the graph, there is a panel that enumerates the node labels and relationship types.

Reviewer: ```suggestion
In addition to the graph, this view also contains the *Results overview* panel that enumerates the node labels and relationship types.
```

If an entity is selected, the panel displays the properties of that entity.

> The panel can be toggled by clicking the button in the top right corner of the graph view. The *Download* button is located to the left of that button and allows you to download the graph as a PNG image.

Reviewer: ```suggestion
The panel can be toggled with the button in the top right corner of the graph view. 
The *Download* button is located to the left of that button and allows you to download the graph as a PNG image.
```

In the bottom right corner of the graph view is a stack of buttons, from top to bottom:

> * *Hierarchy view*—Toggles between force-directed or hierarchical layouts of the graph.
> 
> * *Expand all*—Expands all nodes in the graph, showing all relationships.
> 
> * *Collapse all*—Collapses all nodes in the graph, hiding relationships.
> 
> * *Fit to screen*—Fits the graph to the screen, adjusting the zoom level and centering the graph.

Reviewer: ```suggestion
* *Hierarchy view* — Toggles between force-directed or hierarchical layouts of the graph.

* *Expand all* — Expands all nodes in the graph, showing all relationships.

* *Collapse all* — Collapses all nodes in the graph, hiding relationships.

* *Fit to screen* — Fits the graph to the screen, adjusting the zoom level and centering the graph.
```

== Nodes

The following node labels are used in the graph view.

[cols="1,2"]
|===

> |Label |Description

Author: nit: it would be nice for the values in the first column to look like labels (I think this is used somewhere in Cypher docs). And those in the relationships table like relationships. Should allow quick visual matching...

Reviewer 2: Resolved

|DBMS

> |Represents a Neo4j cluster

Reviewer: As I mentioned elsewhere, this can also represent a single, non-clustered deployment. Maybe I am mistaken, but I am not sure if it is safe to assume that a user's deployment will always be a cluster?
